internal/kubectl/configset.go

```diff
@@ -78,6 +78,7 @@ var setResourceCmd = &cobra.Command{
 // normalizeModeString corrects format of input Synchronization Mode string
 func normalizeModeString(modeStr string) string {
 	low := strings.ToLower(modeStr)
+	//lint:ignore SA1019 we should fix this but not today
 	return strings.Title(low)
```

Comment:
We could consider fixing this staticcheck issue now - just to fix the build (using Go 1.17)

```suggestion
	return cases.Title(language.English).String(low)
```

Reply:
Ok, will do